site/tutorials/gettings_started.md

```diff
@@ -0,0 +1,293 @@
+
+In the visualization that we just crated, Vega-Lite automatically added an axis with labels for the different categories as well as an axis title. However, 3 points are still overlapping in each category. So far, we have only defined a visual encoding for the field `a`. We can add another visual encoding for the field `b`.
+
+```json
+...
+"y": {"field": "b","type": "quantitative"}
+...
+```
+
+This time we set the field type to be `quantitative` because the numeric values in field `b` have a magnitude.
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "point",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal"},
+    "y": {"field": "b", "type": "quantitative"}
+  }
+}
+</div>
+
+Now we can see the raw data points. Note that Vega-Lite automatically added grid lines to the y-axis to help you see the magnitude of the `b` values.
+
+## Aggregate Data
+
+Vega-Lite can do more than just visualizing raw data. By adding `"aggregate": "mean"` to the definition of the y encoding channel, we can see the average value in each category. For example, the average value of category `D` is `(1 + 2 + 6)/3 = 9/3 = 3`.
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "point",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal"},
+    "y": {"field": "b", "type": "quantitative", "aggregate": "mean"}
+  }
+}
+</div>
+
+Great! You computed the aggregate values for each category and visualized the resulting value as a point. Typically aggregated values for categories are visualized using bar charts. To create a bar chart, we have to change the mark type from `point` to `bar`.
+
+
+```diff
+- "mark": "point"
++ "mark": "bar"
+```
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "bar",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal"},
+    "y": {"field": "b", "type": "quantitative", "aggregate": "mean"}
+  }
+}
+</div>
+
+Note that you don't have to specify that you want a vertical bar chart. In fact, we can change the visualization to a horizontal bar chart by swapping the `x` and `y` channels.
```

Comment:
Ok I was a little confused.  Maybe something like: 

> Note that you don't have to specify that you want a vertical bar chart. 

Since the quantitative value is on y, you automatically get a vertical bar chart.  

(I still don't like my sentence, but I guess you get my point.)